packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
+
+        if (bounds === 'bottom') {
+            bundleBadgeLabel = 'From';
+
+            if (price_view !== 'PRICE_RANGE') {
+                bundleBadgeLabel = 'As low as';
+            }
+        }
+
+        if (bounds === 'top') {
+            bundleBadgeLabel = 'To';
+        }
+
+        return (
+            <p
+              mix={ {
+                  block: 'ProductCard',
+                  elem: 'PriceBadge'
+              } }
+            >
+                { __(bundleBadgeLabel) }
+            </p>
+        );
+    }
+
+    renderOldMaxBundlePrice() {
+        const {
+            product: { price_range }
+        } = this.props;
+
+        const oldMaxBundlePrice = `${ price_range.maximum_price.regular_price.value }`;
+        const finalMaxBundlePrice = `${ price_range.maximum_price.final_price.value }`;
+
+        if (finalMaxBundlePrice === oldMaxBundlePrice) {
+            return null;
+        }
+
+        return (
+            <del
+              block="ProductPrice"
+              elem="HighPrice"
+              aria-label={ __('Old product price') }
+            >
+                { roundPrice(`${ oldMaxBundlePrice }`) }
+            </del>
+        );
+    }
+
+    renderMaxBundlePrice() {
+        const {
+            product: { price_range }
+        } = this.props;
+
+        const oldMaxBundlePrice = `${ price_range.maximum_price.regular_price.value }`;
+        const finalMaxBundlePrice = `${ price_range.maximum_price.final_price.value }`;
+
+        return (
+            <p
+              aria-label={ __(`'Maximal product price: ${ finalMaxBundlePrice }'`) }
+                // eslint-disable-next-line react/forbid-component-props
+              className="ProductPrice"
```

Comment:
1. Please don't disable eslint.
2. Please don't use `className` prop, use BEM instead.
https://docs.scandipwa.com/structure/code-style/scss-code-style#avoid-non-bem-selectors

Reply:
In this particular case you already have the `ProductCard-Price` classname. Do you really need to add additional `ProductPrice` classname?
If yes, you can use the `mix` prop.
But i hardly believe you need it here.